store/ARC2_StoreSelectQueryHandler.php

```diff
@@ -1320,6 +1320,9 @@ public function detectExpressionValueType($pattern_ids)
         return '';
     }
 
+    /**
+     * @todo not in use, so remove?
```

Comment:
The method calls are built dynamically in code. IIRC, they come from stuff like "<", "<", "&&", "||" in FILTERs.